baselines/fedmlb/pyproject.toml

```diff
@@ -0,0 +1,140 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "fedmlb" # <----- Ensure it matches the name of your baseline directory containing all the source code
+version = "1.0.0"
+description = "Flower Baselines"
+license = "Apache-2.0"
+authors = ["The Flower Authors <hello@flower.dev>"]
```

Comment:
Update the `authors` fields with your contact details if you'd like. Also, the `description` field above with the title of the paper (probably having just `FedMLB` would do too if you prefer. Up to you!